### DISCUSSION TITLE
能否在版面分析后将表格存储为图片

### DISCUSSION THREAD

**Discussion by WingI9B1** [PaddlePaddle/PaddleOCR#13463] [Q&A]
```
ocr_engine = PPStructure(table=True, ocr=True, show_log=False)
```
现在这样使用代码，识别出表格，结果存储为excel格式。

能否识别为表格，就存储为图片？

**Comment by UserWangZz** [comment]
这需要你自己修改代码

**Reply by WingI9B1** [reply]
要从哪步开始？我只能回溯到`save_structure_res`这块，就是存储阶段。

具体识别到表格的时候是在哪块部分？🙏

**Reply by UserWangZz** [reply]
一般是先使用layout模型进行布局分析，这个阶段应该可以得到对table的检测框，可以在这个阶段对表格图像进行保存

**Reply by GreatV** [reply]
把table那个label的bounding-box crop出来是不是就行了。
